runtime/Cargo.toml

```diff
@@ -107,6 +107,7 @@ assert_matches = { workspace = true }
 ed25519-dalek = { workspace = true }
 libsecp256k1 = { workspace = true }
 memoffset = { workspace = true }
+rand0-7 = { package = "rand", version = "0.7" }
```

Comment:
need this to create the precompile instructions for testing.

logic was mainly copied from the individual precompile tests - if there's a better way without this old version would be happy to change.